edge_computing/image-base-install/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,51 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= About image-based deployments for managed {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} using an image-based installation arrives at a remote site, a technician can easily reconfigure and deploy the host in a matter of minutes.
+
+For clusters with a hub-and-spoke architecture, to complete the deployment of a preinstalled host, you must first define site-specific configuration resources on the hub cluster for each host. These resources contain configuration information such as the properties of the bare-metal host, authentication details, and other deployment and networking information. 
+
+The Image Based Install (IBI) Operator creates a configuration ISO from these resources, and then boots the host with the configuration ISO attached. The host mounts the configuration ISO and runs the reconfiguration process. When the reconfiguration completes, the {sno} cluster is ready.
+
+[NOTE]
+====
+You must create distinct configuration resources for each bare-metal host. 
+====
+
+See the following high-level steps to deploy a preinstalled host in a clusters with a hub-and-spoke architecture:
+
+* Install the IBI Operator on the hub cluster.
+* Create site-specific configuration resources in the hub cluster for each host.
+* Apply these resources in the hub cluster. 
+* The IBI Operator creates a configuration ISO from these resources and boots the target host with the configuration ISO attached.
+* The host mounts the configuration ISO and runs the reconfiguration process. When the reconfiguration completes, the {sno} cluster is ready. 
+
+[NOTE]
+====
+To manually deploy a preinstalled host for a cluster, you must define an `ImageBasedConfig` resource and an installation manifest, and provide these as inputs to the `openshift-install` installation program. For more information see _Deploying a {sno} cluster using the {product-title} installation program_.
```

Comment:
WDYT?
> Alternatively, you manually can deploy a preinstalled host for a cluster without using a hub cluster. You must define an `ImageBasedConfig` resource and an installation manifest, and provide these as inputs to the `openshift-install` installation program. For more information see _Deploying a {sno} cluster using the {product-title} installation program_.

Reply:
Looks good.
Just nit:
Alternatively, you can manually deploy a preinstalled...
Also I think it should use:
openshift-install program.
Instead of "openshift-install installation program" and "{product-title} installation program"